source/pattern-library/data-visualization/heat-map/index.md

```diff
@@ -1,60 +1,8 @@
 ---
-title: Heat Map
-author: dlabrecq
-layout: page-tabs
+author: lhinson
+layout: page-pattern
+overview: patterns/data-visualization/heat-map/overview.md
+design: patterns/data-visualization/heat-map/design.md
+code_html: false
+code_angular: /components/angular-patternfly/dist/docs/partials/api/patternfly.charts.directive.pfHeatMap.html
```

Comment:
1) the _includes folder isn't sourced to the web server directly
2) I tried using a jekyll include instead of the current angular include so we could use the submodule code, but it didn't work.  I plan to track getting that to work with a new story.